app/js/lib/view_handler/animator.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+var objectAssign = require( 'object-assign' ),
+
+	ElementAnimator = {
+		el: null,
+		showElement: function () {
+			throw new Error( 'This is an abstract class!' );
+		},
+		hideElement: function () {
+			throw new Error( 'This is an abstract class!' );
+		}
+	},
```

Comment:
Don't we already have something like this?

Anyway, why did you create this "interface"? Is it "type hinted" against anywhere?